contracts/SponsorFaucet.sol

```diff
@@ -31,36 +32,41 @@ contract SponsorFaucet is Ownable, Pausable, ReentrancyGuard {
         collateral_bound = collateralBound;
         upper_bound = upperBound;
     }
-    
+     
     /*** Dapp dev calls ***/ 
     //apply cfx for gas & storage
     function applyFor(address dapp) public nonReentrant whenNotPaused {
         //todo: internal contract require check
-        cpc.set_sponsor_for_gas.value(gas_bound)(dapp, upper_bound);
-        cpc.set_sponsor_for_collateral.value(collateral_bound)(dapp);
-        dapps[dapp].cnt.add(1);
+        cpc.setSponsorForGas.value(gas_bound)(dapp, upper_bound);
+        cpc.setSponsorForCollateral.value(collateral_bound)(dapp);
+        dapps[dapp].gas_cnt.add(1);
+        dapps[dapp].collateral_cnt.add(1);
         dapps[dapp].limit.add(gas_bound.add(collateral_bound));
         emit applied(msg.sender, dapp, upper_bound);
```

Comment:
could reuse `applyForGas` and `applyForCollateral`

Reply:
Better not. If reuse, the modifier in **applyForGas** and **applyForCollateral** will cause extra gas cost.